purchase_request_type/i18n/purchase_request_type.pot

```diff
@@ -99,7 +99,6 @@ msgid "Name"
 msgstr ""
 
 #. module: purchase_request_type
-#: model:ir.model.fields,field_description:purchase_request_type.field_purchase_request__request_type
 #: model_terms:ir.ui.view,arch_db:purchase_request_type.view_purchase_request_search
 msgid "PR Type"
```

Comment:
It seems this is still referred in one place, so it should be changed also, isn't it?